BUILD.md

```diff
@@ -0,0 +1,48 @@
+# Building the RABET-V Documentation
+
+## Dependencies
+
+In order to build the RABET-V Documentation, the following components must be installed:
+
+- Python 3+
+- [Pipenv](https://pipenv.pypa.io/en/latest/)
+- pandoc 1.16
+- [Imagemagick](https://imagemagick.org/index.php)
+- [Inkscape](https://inkscape.org/)
```

Comment:
Added Inkscape as a requirement. In the end, we'll only need Imagemagick or Inkscape, but not both. Just depends whether it's easier/better to convert the SVGs in Sphinx or pandoc. Since HTML can display SVGs without issue, I'd rather do it in pandoc.

Reply:
I had issues with Inkscape. It seems to be omitting the text in the SVGs. Can you validate?